packages/kotti-ui/source/kotti-navbar/components/NavbarQuickLink.vue

```diff
@@ -1,53 +1,74 @@
 <template>
-	<div class="kt-navbar-quick-links">
-		<div v-if="!isNarrow" class="kt-navbar-quick-links__title" v-text="title" />
+	<div class="kt-navbar__quick-links">
+		<div
+			v-if="!isNarrow"
+			class="kt-navbar__quick-links__title"
+			v-text="title"
+		/>
 		<a
 			v-for="(item, index) in links"
 			:key="index"
-			class="kt-navbar-quick-links__link"
+			class="kt-navbar__quick-links__link"
 			:href="item.link"
 			rel="noopener noreferrer"
 			target="_blank"
 		>
-			<span v-if="!isNarrow" v-text="item.title" /> <i class="yoco">link</i>
+			<span v-if="!isNarrow" v-text="item.title" />
+			<i class="yoco" v-text="Yoco.Icon.LINK" />
 		</a>
 	</div>
 </template>
 
-<script>
-export default {
+<script lang="ts">
+import { Yoco } from '@3yourmind/yoco'
+import { computed, defineComponent } from '@vue/composition-api'
+
+import { useTranslationNamespace } from '../../kotti-translation/hooks'
+import { Kotti } from '../../types'
+
+export default defineComponent<{
+	links: Kotti.Navbar.QuickLink[]
+}>({
 	name: 'KtNavbarQuickLink',
 	props: {
-		title: { type: String, default: 'Quick Links' },
-		links: { type: Array, default: () => [] },
+		links: { required: true, type: Array },
 	},
-	computed: {
-		isNarrow() {
-			return this.$KtNavbar.isNarrow
-		},
+	setup(props, { root }) {
+		const translations = useTranslationNamespace('KtNavbar')
+
+		return {
+			// @ts-expect-error
```

Comment:
why tho

Reply:
Because of the legacy `GlovalVue.$KtNavbar` hacks

Reply:
then please add a message description